game/src/debug/objects.rs

```diff
@@ -20,6 +20,12 @@ impl ObjectDebugger {
                     "cam_x = {}, cam_y = {}",
                     g.canvas.cam_x, g.canvas.cam_y
                 ));
+                if let Some(ID::Lane(l)) = app.primary.current_selection {
+                    let pl = &app.primary.map.get_l(l).lane_center_pts;
+                    if let Some((dist, _)) = pl.dist_along_of_point(pl.project_pt(pt)) {
+                        txt.add_line(Position::new(l, dist).to_string());
```

Comment:
If you hold left control in debug mode over a lane, you get the dist_along on that lane. Super useful as I was figuring out `equiv_pos` bugs here.